modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/artifact/MavenClassLoaderModelLoaderTestCase.java

```diff
@@ -57,20 +55,39 @@ public void before() {
 
   @Test
   public void noMavenConfiguration() throws Exception {
+    Map<String, String> properties = getMuleFreeSystemProperties();
+    GlobalConfigLoader.reset(); //Change local repository path
```

Comment:
I will move this to the block inside the withSysteProperties but set the MULE_RUNTIME_CONFIG_MAVEN_REPOSITORY_LOCATION system property in the Map of properties to be used so you warranty that the localRepositoryLocation is a temp location when the test runs